src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -131,6 +131,21 @@ namespace OpenRCT2::Scripting
             return GameIsPaused();
         }
 
+        void paused_set(const bool& value)
+        {
+#    ifndef DISABLE_NETWORK
+            if (NetworkGetMode() != NETWORK_MODE_NONE)
+            {
+                auto ctx = GetContext()->GetScriptEngine().GetContext();
+                duk_error(
+                    ctx, DUK_ERR_ERROR, "Setting paused state is not network safe. Use the pausetoggle game action instead.");
+                return;
+            }
+#    endif
```

Comment:
Is there a reason to not use `ThrowIfGameStateNotMutable()` that all other setters use to check for unsafe state modifications?

Reply:
I didn't know about that method, I will check it out

Reply:
Wouldn't I know it but the method does exactly what I programmed. It is implemented.